### ISSUE TITLE
How do i spawn a urdfpawn? 

### ISSUE THREAD

**Issue by ThomasR155** [mitchellspryn/UrdfSim#50]
I followed the documentation and tried to integrate lunabot into my Project.
Do I need to spawn a pawn in UE myself? or do i just need a player start actor.

**Comment by mitchellspryn** [comment]
No, you should just need to set a PlayerStart and set the mode to "AirSimGameMode"
